src/intent.ts

```diff
@@ -0,0 +1,33 @@
+/*
+* Intent schema
```

Comment:
This isn't a schema, it is an interface ;-)

This would be a good place to describe what an intent actually is in 'plain english'